contributors/guide/help-wanted.md

```diff
@@ -21,41 +22,36 @@ grow and improve our community.
 
 ## Help Wanted
 
-Items marked with the `help wanted` label need to ensure that they are:
+Items marked with the `help wanted` label need to ensure that they meet these criteria:
 
 - **Low Barrier to Entry**
-
-  It should be tractable for new contributors. Documentation on how that type of
+  It should be easy for new contributors. Documentation on how that type of
   change should be made should already exist.
 
 - **Clear Task**
-
   The task is agreed upon and does not require further discussions in the
   community. Call out if that area of code is untested and requires new
   fixtures.
 
-  API / CLI behavior is decided and included in the OP issue, for example: _"The
-  new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`"_, with
+  API and CLI behavior should be decided and included in the OP issue, for example: "The
+  new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`", with
   expected validations called out.
 
-- **Goldilocks priority**
-
-  Not too high that a core contributor should do it, but not too low that it
-  isn't useful enough for a core contributor to spend time to review it, answer
-  questions, help get it into a release, etc.
+- **[Goldilocks](https://www.gutenberg.org/files/49001/49001-h/49001-h.htm#Page_207) priority**
+  The priority should not be so high that a core contributor should do it, but not too low that it
+  isn't useful enough for a core contributor to spend time reviewing it, answering
+  questions, helping get it into a release, etc.
 
 - **Up-To-Date**
+  Often these issues become obsolete and have already been completed, are no longer
+  desired, no longer make sense, or have changed priority or difficulty.
 
-  Often these issues become obsolete and have already been done, are no longer
-  desired, no longer make sense, have changed priority or difficulty , etc.
-
-Example of a Help Wanted issue description can be found here: [kubernetes/test-infra#21356 (comment)](https://github.com/kubernetes/test-infra/issues/21356#issuecomment-799972711).
+A good example of a Help Wanted issue description can be found here: [kubernetes/test-infra#21356 (comment)](https://github.com/kubernetes/test-infra/issues/21356#issuecomment-799972711).
 
-Related commands:
+These commands can be used with GitHub issues to manage the `help wanted` label:
 
 - `/help` : Adds the `help wanted` label to an issue.
-- `/remove-help` : Removes the `help wanted` label from an issue. If the
-  `good first issue` label is present, it is removed as well.
+- `/remove-help` : Removes the `help wanted` label from an issue. If the `good first issue` label is present, it is removed as well.
```

Comment:
I think the content here is identical. In this instance, the original is more aligned with our style guide as it keeps it at 80 characters.